.travis.yml

```diff
@@ -14,7 +14,7 @@ sudo: false  # use the new container-based Travis infrastructure
 dist: trusty # Ubuntu 14.04
 
 mono:
-  - latest
+  - nightly
```

Comment:
Problem is I don't think there is one ATM. I tried reverting but we need msbuild15 to build now.. So we would need to install that on travis and I was too lazy to set everything up - not knowing if it would even work.

Lets hope the nightly works ;)